journal/original/32-epistles.adoc

```diff
@@ -1,8 +1,9 @@
 == Epistles and Papers of William Edmundson.
 
-An epistle to Friends, written in Jamaica.
-1671.
+[.signed-section-context-open]
+An epistle to Friends, written in Jamaica. 1671.
```

Comment:
Maija, I won't re-make all of the same comments on the original edition, hopefully you can just apply all of the corrections to both versions like normal. Thanks!